docs/source/software/r.rst

```diff
@@ -28,3 +29,71 @@ You can then install R packages that you need. For example, to install a package
    if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
    BiocManager::install("rtracklayer")
+
+**When using Rstudio in the OOD App:**
+
+1. Launch an RStudio instance on the OOD.
+2. In the Rstudio console type:
+
+         ``install.packages("packrat)`` 
+
+.. note::
+        This will install by default in ``$HOME/R/x86_64-pc-linux-gnu-library/<version>/`` as long as you don't have previous environments or those have been turned off (see below). For packrat installation, it is best to specify a "project folder" in your $HOME, /scratch or /work directory (if you do not have /work please see here for access). The location /tmp/Rtmp8CbQCA/downloaded_packages would not work because /tmp corresponds to the compute node that you were on while running the R session. Optimally, you would like to have the packrat location in a persistent place so that all packages and libraries are available to you at all times regardless of the compute node you are on. 
+
+3. In the Rstudio terminal (the second tab in the lower left quad) create the packrat project directory at the desired location using ``mkdir``. Or you can select "New Folder" in the "Files" tab in the lower right hand side.
```

Comment:
I don't know if you should move the mkdir command up to be right after "...using ``mkdir``' and then after that started the "Or you can select...".

Reply:
Yeah I just changed the order of the sentences so they're told how to use mkdir last right before the command example.